### PR TITLE
chore: Update merge main script to handle the renaming of the base tsconfig.x.json files

### PR DESCRIPTION
**Pull request by MSNev**
chore: Update merge main script to handle the renaming of the base files
- Required as last merged otel/js and otel/contrib-js renamed/moved the tsconfig.es5.json files.

chore: Add resources package to merge to main

chore: Add context-zone packages to merge to main script

### CODE REVIEW

File: karma.debug.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const LOG_DISABLE = "OFF";
 process.env.CHROME_BIN = require("puppeteer").executablePath();
 
 // Default to using edge locally -- choose your own browser as required
-process.env.CHROME_BIN = "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe";
+// process.env.CHROME_BIN = "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe";
```
Comment (MSNev) [inline karma.debug.js:29]: Not everyone has windows with Edge 😄

File: sandbox-tools/merge-repos/src/config.ts [hunk 2]
```diff
@@ -109,10 +109,10 @@ export const reposToSyncAndMerge: IRepoSyncDetails = {
...
-    // { name: "@opentelemetry/context-zone", destPath: "pkgs/context/zone/", srcPath: "auto-merge/js/packages/opentelemetry-context-zone/", bundleName: "otel-sndbx.context-zone", bundleNamespace: "opentelemetry.sandbox.web.context-zone", noTests: true },
-    // { name: "@opentelemetry/context-zone-peer-dep", destPath: "pkgs/context/zone-peer-dep/", srcPath: "auto-merge/js/packages/opentelemetry-context-zone-peer-dep/", bundleName: "otel-sndbx.context-zone-peer-dep", bundleNamespace: "opentelemetry.sandbox.web.context-zone-peer-dep", noWorkerTests: true, noNodeTests: true },
+    { name: "@opentelemetry/core", destPath: "pkgs/core/", srcPath: "auto-merge/js/packages/opentelemetry-core/", bundleName: "otel-sndbx.core", bundleNamespace: "opentelemetry.sandbox.web.core" },
+    { name: "@opentelemetry/resources", destPath: "pkgs/resources/", srcPath: "auto-merge/js/packages/opentelemetry-resources/", bundleName: "otel-sndbx.resources", bundleNamespace: "opentelemetry.sandbox.web.resources" },
```
Comment (MSNev) [inline sandbox-tools/merge-repos/src/config.ts:113]: Adding additional packages to the auto-merge

File: sandbox-tools/merge-repos/src/mergeStagingToMain.ts [hunk 3]
```diff
@@ -343,6 +343,10 @@ async function getStagingRepo(git: SimpleGit, repoName: string, details: IRepoDe
 
         let src = path.join(forkDestOrg, fileDetails.srcPath).replace(/\\/g, "/");
         if (!fs.existsSync(src)) {
+            if (fileDetails.optional) {
```
Comment (MSNev) [inline sandbox-tools/merge-repos/src/mergeStagingToMain.ts:346]: Don't fail to script if the file being copied is marked as "optional"

**Comment by MSNev** [comment]
Example PR from these changes #53, this is larger for 2 reasons
- The last merge js/contrib -> auto-merge/repo-staging branch included changes from js/contrib which have not yet been merged to `main` because of the `tsconfig.es5.json` removal / reworking
- This now includes the 3 additional package moves.

**Review by martinkuba — APPROVED** [review]

**Pull request merged**